pypit/arqa.py

```diff
@@ -514,6 +521,16 @@ def plot_orderfits(slf, model, ydata, xdata=None, xmodl=None, textplt="Slit", ma
     maskval : float, (optional)
       Value used in arrays to indicate a masked value
     """
+    # Outfil
+    module = inspect.stack()[0][3]
+    if 'Arc' in desc:
+        module += '_Arc'
+    elif 'Blaze' in desc:
+        module += '_Blaze'
+    else:
+        msgs.error("Should not get here")
```

Comment:
Maybe use a slightly more informative error?

Or, perhaps this should raise a `msgs.bug` instead?